docs/configuration.md

```diff
@@ -139,6 +139,7 @@ Iceberg catalogs support using catalog properties to configure catalog behaviors
 | clients                           | 2                  | client pool size                                       |
 | cache-enabled                     | true               | Whether to cache catalog entries |
 | cache.expiration-interval-ms      | 30000              | How long catalog entries are locally cached, in milliseconds; 0 disables caching, negative values disable expiration |
+| metrics-reporter-impl | org.apache.iceberg.metrics.LoggingMetricsReporter | a custom `MetricsReporter` implementation to use in a catalog |
```

Comment:
Link back to the `metrics-reporting` page

Reply:
Another thing I just wanted to point this out is we should always use [sentence style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization) for new sentences.

I'll address this in an overall fix later but feel free to change it if you're already fixing adding the link.

Reply:
I was trying to be consistent with the existing descriptions in this table